DOCUMENTS/POSTFIX_CONF.txt

```diff
@@ -13,12 +13,19 @@ postconf -m)  Three main.cf variables are involved:
 
 virtual_mailbox_domains = proxy:mysql:/etc/postfix/sql/mysql_virtual_domains_maps.cf
 virtual_alias_maps =
-   proxy:mysql:/etc/postfix/sql/mysql_virtual_alias_maps.cf,
```

Comment:
There is a reason why mysql_virtual_alias_maps.cf is listed first - an alias domain can "override" some aliases. To keep this working, mysql_virtual_alias_maps.cf needs to be the first in the list.

Why did you move it around?